examples/kind/README.md

```diff
@@ -0,0 +1,72 @@
+# KIND Demo
+
+[kind](https://github.com/kubernetes-sigs/kind)(Kubernetes in Docker) is a tool for running local Kubernetes clusters using Docker container “nodes”. Azure CSI Driver will work in kind using Service Principal.
+
+## Prerequisite
+
+- Follow [kind installation instruction](https://github.com/kubernetes-sigs/kind#installation-and-usage) to setup kind in your machine
+
+> Windows 10 users could you wsl 2 to install kind and run this sample. Integrate docker for windows with wsl 2 by following the [instructions](https://docs.docker.com/docker-for-windows/wsl-tech-preview/).
+
+## Setup
+
+- Create a Azure AD App to create Service Principal and give it "GET" permission for secrets in keyvault. Follow the steps in [keyvault docs](https://docs.microsoft.com/en-us/azure/key-vault/general/group-permissions-for-apps#applications). Keep `ClientID` and `ClientSecret` of the AD app handy.
```

Comment:
maybe you can link to this https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/docs/service-principal-mode.md